src/components/Modals/FiatRamps/FiatRampsCommon.ts

```diff
@@ -10,17 +10,12 @@ export enum FiatRampAction {
   Sell = 'sell',
 }
 
-export type FiatRampCurrency = {
+export type FiatRampAsset = {
   name: string
   assetId: string
   symbol: string
   imageUrl?: string
   disabled?: boolean
+  cryptoBalance?: BigNumber
+  fiatBalance?: BigNumber
```

Comment:
In this way we won't be able to make discriminations based on the balance properties at all, as opposed to previous implementation: these two properties will either never be there at all, or be present together, by just making them optional, we'll be allowed to pass none (which is correct) or just one or the other (which is wrong)

Maybe something like so ([TypeScript Playground](https://www.typescriptlang.org/play?#code/C4TwDgpgBAKhDOwoF4oG8BQVtQGYHt8AuKRAJwEsA7AcyxwCMBDMk86ugXygDIoAKTDijMAXm2CVa9bAEcArgA8JUrlAA+6GSKaiA-CSoQAbhDLaFig1COnznAJQYMAY3xVEUYAmAk4n1CEcAmIoAHIAd3wyAGt4MIAabWZWcKjY+KThMRIwgBN8BBt8JHSYsIxOZzcPJG9EGCi-HxQtYRDcsszklk7ouMSe8TT+7uFLPoyKziA))? Note, there might be a cleaner way to achieve this with the right generics